definitions/infra-container/golden_metrics.yml

```diff
@@ -12,7 +12,7 @@ cpuUtilization:
     newRelic:
       select: max(docker.container.cpuPercent) or max(k8s.container.cpuCoresUtilization) AS 'CPU Utilization (%)'
     opentelemetry:
-      select: max(container.cpu.percent) AS 'CPU Utilization (%)'
+      select: max(container.cpu.utilization) AS 'CPU Utilization (%)'
```

Comment:
This metric was updated to match the new metric used in the collector after [v0.79.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.79.0). `container.cpu.percent` was deprecated in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/22823 to match opentelemetry naming conventions.